README.md

```diff
@@ -4,6 +4,13 @@
 
 SecureDrop is an open-source whistleblower submission system that media organizations can use to securely accept documents from and communicate with anonymous sources. It was originally created by the late Aaron Swartz and is currently managed by [Freedom of the Press Foundation](https://freedom.press).
 
+## Found an issue?
+
+If you're here because you want to report an issue in SecureDrop, please observe the following protocol to report an issue responsibly:
+
+* If you want to report a **_security issue_**, please use our [Bugcrowd](https://bugcrowd.com/freedomofpress).
```

Comment:
nit... instead of `please use our [Bugcrowd]` for people who don't know bugcrowd would say `please use our bug bounty hosted by [Bugcrowd]` or something similar.